client/libs/material/src/lib/material.module.ts

```diff
@@ -44,11 +47,21 @@ const MODULES = [
   MatDatepickerModule,
   MatNativeDateModule,
   MatMenuModule,
+  MatSortModule,
+  MatPaginatorModule,
   MatDialogModule
 ];
 
 @NgModule({
-  imports: MODULES,
-  exports: MODULES
+  imports: [
+    MODULES,
+    MatSortModule,
```

Comment:
Add `MatSortModule` and `MatTableModule` to the above `MODULES` array.

Also, the `imports` array should not contain an array of array's. Just leave the `MODULES` here.